src/util.js

```diff
@@ -24,6 +24,20 @@ var Util = {
         total: 0,
         message: null
     },
+    
+    countOccurences: function(contentString, searchString) {
```

Comment:
Thanks.  I should probably turn spell check back on and just add programming jargon to the dictionary file.